OpenRA.Mods.RA/AI/AttackOrFleeFuzzy.cs

```diff
@@ -14,6 +14,7 @@
 using AI.Fuzzy.Library;
 using OpenRA.Mods.RA.Traits;
 using OpenRA.GameRules;
+using OpenRA.Mods.RA.Move;
```

Comment:
This is probably a merge conflict edited towards the wrong direction. Remove it again.